lib/WebSocketTransport.js

```diff
@@ -25,7 +25,10 @@ class WebSocketTransport {
    */
   static create(url) {
     return new Promise((resolve, reject) => {
-      const ws = new WebSocket(url, [], { perMessageDeflate: false });
+      const ws = new WebSocket(url, [], {
+        perMessageDeflate: false,
+        maxPayload: 256 * 1024 * 1024, // 256Mb
```

Comment:
why is there a max payload at all? What would happen if there was no max and someone tried to send 8 gb?

Reply:
I guess to protect against process crashes.